docs/assets/css/main.css

```diff
@@ -18720,3 +18722,34 @@ footer {
     background: rgba(5, 165, 146, 0.3);
     margin-top: 10px;
   }
+
+/*image-thumbnail-grid*/
+  
+.row {
+    display: flex;
+  }
+  
+.column {
+    flex: 25.00%;
+    padding: 5px;
+  }
+
+.img-thumbnail {
```

Comment:
The book cover image is skewed.
<img width="258" alt="Screen Shot 2020-10-22 at 11 23 23 AM" src="https://user-images.githubusercontent.com/7570704/96901609-18768f00-1459-11eb-8bff-69b08bbf6490.png">

Reply:
Created https://github.com/layer5io/meshery/issues/1763 to track this required change.